projects/swimlane/ngx-ui/src/lib/utils/coerce/coerce-number.ts

```diff
@@ -0,0 +1,16 @@
+import { coerceNumberProperty } from '@angular/cdk/coercion';
+
+export function CoerceNumberProperty(fallback?: number): PropertyDecorator {
+  return function (target, propertyKey) {
+    const _key = Symbol();
+    target[_key] = target[propertyKey];
+    Object.defineProperty(target, propertyKey, {
+      get(): number {
+        return this[_key];
+      },
+      set(v: any) {
```

Comment:
nitpick: I think the typing should be `unknown` instead of `any` and using descriptive variable names instead of `v` using `value` would be preferred.